src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs

```diff
@@ -163,7 +163,7 @@ private static NewArrayExpression WrapParametersInObjectArray(ParameterExpressio
 
         private class DelegateCallInterceptedEventRaiser
         {
-            public static readonly MethodInfo RaiseMethod = typeof(DelegateCallInterceptedEventRaiser).GetMethod("Raise");
+            public static readonly MethodInfo RaiseMethod = typeof(DelegateCallInterceptedEventRaiser).GetMethod(nameof(Raise));

```

Comment:
Mostly I wanted the change so I'd stop getting complaints that `Raise` wasn't used.